Corona-Warn-App/src/main/res/layout/fragment_submission_your_consent.xml

```diff
@@ -0,0 +1,158 @@
+
+                <LinearLayout
+                    android:id="@+id/submission_your_consent_agreement_card"
+                    style="@style/cardTracing"
+                    android:layout_width="@dimen/match_constraint"
+                    android:layout_height="wrap_content"
+                    android:layout_marginTop="@dimen/spacing_medium"
+                    android:focusable="true"
+                    android:orientation="vertical"
+                    app:layout_constraintEnd_toStartOf="@+id/guideline_card_end"
+                    app:layout_constraintStart_toStartOf="@+id/guideline_card_start"
+                    app:layout_constraintTop_toBottomOf="@+id/submission_your_consent_about_text">
+
+                    <TextView
+                        android:id="@+id/submission_your_consent_agreement_title"
+                        style="@style/headline6"
+                        android:layout_width="match_parent"
+                        android:layout_height="wrap_content"
+                        android:layout_marginEnd="@dimen/spacing_small"
+                        android:accessibilityHeading="true"
+                        android:contentDescription="@string/submission_your_consent_agreement_title"
+                        android:text="@string/submission_your_consent_agreement_title"/>
+
+                    <TextView
+                        android:id="@+id/submission_your_consent_agreement_share_test_results_text"
+                        style="@style/subtitle"
+                        android:layout_width="match_parent"
+                        android:layout_height="wrap_content"
+                        android:layout_marginTop="@dimen/spacing_normal"
+                        android:text="@string/submission_your_consent_agreement_share_test_results"/>
+
+                    <de.rki.coronawarnapp.ui.view.CountryListView
+                        android:id="@+id/submission_your_consent_agreement_country_list"
+                        android:layout_width="match_parent"
+                        android:layout_height="wrap_content"
+                        android:layout_marginTop="@dimen/spacing_small" />
+
+                    <TextView
+                        android:id="@+id/submission_your_consent_agreement_share_symptoms_text"
+                        style="@style/subtitle"
+                        android:layout_width="match_parent"
+                        android:layout_height="wrap_content"
+                        android:layout_marginTop="@dimen/spacing_normal"
+                        android:text="@string/submission_your_consent_agreement_share_symptoms"/>
+
+                </LinearLayout>
+
+                <View
+                    android:id="@+id/submission_your_consent_agreement_details_divider_top"
+                    android:layout_width="@dimen/match_constraint"
+                    android:layout_height="@dimen/card_divider"
```

Comment:
For whatever reason this divider and the bottom one have 1.48px height in the figma design. Now that I look at it, all  of the dividing lines from the design have different heights...